### ISSUE TITLE
create ForC interactive map

### ISSUE THREAD

**Issue by teixeirak** [forc-db/ForC#125]
Let's create an interactive map (e.g., Google map) with all ForC sites. @ValentineHerr thinks this could be done through R, which is probably the best way to go (for automatic updating). Fusion tables and MyMaps are also possibilities.

We can also  look into the possibility of getting ForC on [Smithsonian's GIS page](https://si.maps.arcgis.com/home/index.html).

**Comment by mcgregorian1** [comment]
I will upload the .kml file plus the script in a second.

I did a quick check with doing a .kml in GoogleMaps and it seems Google got rid of that compatibility. Plus, as Krista saw, they put a hard cap of 2000 sites for the MyMaps thing.

That being said, the .kml can be easily viewed in the [web viewer](https://earth.google.com/web/) of Google Earth in addition to the desktop. All someone has to do is import the .kml file from their local copy of Github and everything appears as we saw in the desktop version.

I can make a separate README for this maybe if you want?

**Comment by teixeirak** [comment]
sure, that would be helpful. In that case, lets create a separate folder to hold the .klm and readme.

**Comment by mcgregorian1** [comment]
I can do that, but I've been unable to make the point headers as we want them. After reading below, did you still want me to do that?

After looking at it again, I realized the issue now is trying to have all the columns displayed for each point with an overarching label that is sites.sitename. Multiple searches have hit brick walls since the dataset_options argument of the writeOGR command is labeled as "experimental."

- alternatively, loading MapView package provides an interactive map that we could then publish as an html. This would skip the .kml file use.

**Comment by mcgregorian1** [comment]
I've updated the R-script and I've added the html file in Github. The map is now viewable in a browser and has labels that reflect the sites.sitename column. The script has options for making the map better if want in the future (the mapview function is very customizable).

I've asked Bill Fisher if he can help with downloading a basemap that is Google satellite. I'll make changes based on what he advises.

I've deleted the .kml file.
